ansible/vars/topo_wan-4link.yml

```diff
@@ -0,0 +1,102 @@
+topology:
+  topo_type: wan
+  VMs:
+    ARISTA01T1:
+      vlans:
+        - 0
+        - 1
+        - 2
+        - 3
+      vm_offset: 0
+    ARISTA02T1:
+      vlans:
+        - 10
+        - 11
+        - 12
+        - 13
+      vm_offset: 1
+  DUT:
+    loopback:
+      ipv4:
+        - 10.1.0.32/32
+      ipv6:
+        - FC00:1::32/128
+wan_dut_configuration:
+  vlab-01:
+    interfaces:
+      PortChannel101:
+        intfs: [0, 1, 2, 3]
+        ipv4: 10.0.0.56/31
+        ipv6: FC00::71/126
+      PortChannel102:
+        intfs: [10, 11, 12, 13]
+        ipv4: 10.0.0.58/31
+        ipv6: FC00::75/126
+configuration_properties:
+  common:
+    dut_asn: 65100
+    dut_type: InternetBackboneRouter
+    swrole: leaf
```

Comment:
Is the role still `leaf`? Could it be something like `backbone` or even hub?

Reply:
change swrole from leaf to core and dut_type to Router.